pkg/collector/corechecks/system/memory_windows.go

```diff
@@ -25,17 +28,76 @@ var runtimeOS = runtime.GOOS
 // MemoryCheck doesn't need additional fields
 type MemoryCheck struct {
 	core.CheckBase
+	cacheBytes     *pdhutil.PdhCounterSet
+	committedBytes *pdhutil.PdhCounterSet
+	pagedBytes     *pdhutil.PdhCounterSet
+	nonpagedBytes  *pdhutil.PdhCounterSet
 }
 
 const mbSize float64 = 1024 * 1024
 
+// Configure handles initial configuration/initialization of the check
+func (c *MemoryCheck) Configure(data integration.Data, initConfig integration.Data) (err error) {
+	c.cacheBytes, err = pdhutil.GetCounterSet("Memory", "Cache Bytes", "", nil)
+	if err == nil {
+		c.committedBytes, err = pdhutil.GetCounterSet("Memory", "Committed Bytes", "", nil)
+		if err == nil {
+			c.pagedBytes, err = pdhutil.GetCounterSet("Memory", "Pool Paged Bytes", "", nil)
+			if err == nil {
+				c.nonpagedBytes, err = pdhutil.GetCounterSet("Memory", "Pool Nonpaged Bytes", "", nil)
+			}
+		}
+	}
+	return err
+}
+
 // Run executes the check
 func (c *MemoryCheck) Run() error {
 	sender, err := aggregator.GetSender(c.ID())
 	if err != nil {
 		return err
 	}
 
+	var val float64
+	if c.cacheBytes != nil {
+		val, err = c.cacheBytes.GetSingleValue()
+		if err == nil {
+			sender.Gauge("system.mem.cached", float64(val)/mbSize, "", nil)
+		} else {
+			log.Warnf("Could not retrieve value for system.mem.cached %v", err)
+			return err
```

Comment:
I think we'd want to attempt to collect the other metrics even if for some reason we encountered an error on this particular metric. Maybe simply removing the `return` statement is enough...

(applies to the other metrics as well)